app/helpers/hyku_addons/revised_admin_set_work_form_helper.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module HykuAddons
+  module RevisedAdminSetWorkFormHelper
```

Comment:
How about renaming this?
```suggestion
  module SimplifiedAdminSetSelectionWorkFormHelper
```